README.md

```diff
@@ -27,9 +27,10 @@ complete example.
 {
   description = "raspberry-pi-nix example";
   nixConfig = {
-    extra-substituters = [ "https://raspberry-pi-nix.cachix.org" ];
+    extra-substituters = [ "https://raspberry-pi-nix.cachix.org" https://adminy.cachix.org ];
```

Comment:
Is this needed upstream? We should commit on one binary-cache I would argue

Reply:
No problem. I removed it. for anyone wanting a cached kernel build, you can now get it from the previous commit :)